framework/auth/campaigns.py

```diff
@@ -69,6 +69,17 @@ def get_campaigns():
                     }
                 })
 
+            # Proxy campaigns: Registries, OSF only
+            newest_campaigns.update({
+                'osf-registries': {
```

Comment:
I hard-code the campaign here since there is no osf model for registries providers yet.